controller/src/beerocks/master/db/db.cpp

```diff
@@ -3600,3 +3600,20 @@ void db::clear_vap_list()
 }
 
 const std::shared_ptr<db::vaps_list_t> db::get_vap_list() { return m_vap_list; }
+
+bool db::is_prplmesh(const std::string &mac)
+{
+    auto node = get_node(mac);
+    if (!node) {
+        LOG(ERROR) << "can't find node with mac " << mac << ", consider as not prplmesh";
+        return false;
+    }
+    return node->is_prplmesh;
+}
+void db::set_prplmesh(const std::string &mac)
+{
+    if (!get_node(mac)) {
+        add_node(network_utils::mac_from_string(mac));
```

Comment:
I don't think a new node should be added in this case. The type of the node is going to be wrong. When we implement #524 we will need to know which type of node to create, so this isn't even possible.

Reply:
I think we create the node only in during the autoconfig M1, so the node will always be created first when we get the autoconfig search request arrives, and if we don't create it here it we won't mark it as prplmesh when we mean to.
@vitalii-komisarenko please confirm.